peter-piper.js

```diff
@@ -22,4 +22,31 @@ var counter = 0;
 // TODO: Count the number of times an uppercase P or lowercase p character
 //       occurs in the `peterpiper` string.
 
+
+peterpiper = peterpiper.toLowerCase();
+
+for (var i = 0; i < peterpiper.length; i++) {
+    if (peterpiper.charAt(i) == 'p') {
+        counter++;
+    }
+}
+
 console.log("There are " + counter + " P's in Peter Piper.");
+
+
+/* Extra Credit
+function countOccurences(myString, myChar) {
+    myString = myString.toLowerCase();
```

Comment:
Currently the `countOccurrences` function is using the `counter` variable that's outside of the function. This will cause issues if you call `countOccurrences` multiple times.

`countOccurrences` should declare it's own variable (`myCounter` or something similar), increment it when a match is found, and return `myCounter` at the end instead of logging a message. This makes `countOccurrences` more reusable.

You should be able to use `countOccurrences` like so:
```js
var counter = countOccurrences(peterpiper, 'p');
console.log("There are " + counter + " P's in Peter Piper.");
```